deps/media-playback/media-playback/decode.c

```diff
@@ -336,6 +336,15 @@ bool mp_decode_next(struct mp_decode *d)
 					d->stream->time_base,
 					(AVRational){1, 1000000000});
 
+		
+		d->frame_pts = av_rescale_q(d->frame_pts,
+			(AVRational) {1, d->m->speed_percentage},
+			(AVRational) {1, 100});
+
+		duration = av_rescale_q(duration,
+				(AVRational) {1, d->m->speed_percentage},
+				(AVRational) {1, 100});
```

Comment:
Style:  Don't put a space between (AVRational) and the opening curly brace after it.